tests/unit/event/event_core.js

```diff
@@ -25,12 +24,9 @@ QUnit.module( libName, {
 		// the collections existence in non touch enabled test browsers
 		$.Event.prototype.touches = [ { pageX: 1, pageY: 1 } ];
 
-		$.mobile.pageContainer = originalPageContainer || $( "body" );
-
-		$( $.mobile.pageContainer ).unbind( "throttledresize" );
+		$( ".ui-pagecontainer" ).unbind( "throttledresize" );
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?